services/user/Tokens/ui/src/components/transfer-modal.tsx

```diff
@@ -0,0 +1,100 @@
+import {
+  AlertDialog,
+  AlertDialogAction,
+  AlertDialogCancel,
+  AlertDialogContent,
+  // AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+} from "@/components/ui/alert-dialog";
+import { useChainId } from "@/hooks/network/useChainId";
+import { useLoggedInUser } from "@/hooks/network/useLoggedInUser";
+import { useBalances } from "@/hooks/tokensPlugin/useBalances";
+import { createIdenticon } from "@/lib/createIdenticon";
+import { Quantity } from "@/lib/quantity";
+
+interface Props {
+  open: boolean;
+  onContinue: () => void;
+  onClose: () => void;
+  isPending: boolean;
+  title?: string;
+  from?: string | null;
+  amount?: string;
+  tokenId?: number;
+  to?: string;
+}
+
+export const TransferModal = ({
+  open,
+  onContinue,
+  onClose,
+  from,
+  to,
+  isPending,
+  tokenId,
+  amount,
+}: Props) => {
+  const { data } = useChainId();
+  const chainId = data || "";
+
+  const { data: currentUserData } = useLoggedInUser();
+
+  const { data: balances } = useBalances(currentUserData);
+  const token = balances?.tokens.find((token) => token.id == tokenId);
+
+  const convertedToInteger =
+    Number(amount) * Math.pow(10, token?.precision || 0);
+
+  const quantity = token
+    ? new Quantity(convertedToInteger.toString(), token.precision, token.id)
+    : undefined;
+
+  const numSymbol = quantity ? quantity.format(true).split(" ") : ["", ""];
+  const [amountFormatted, symbol] = numSymbol;
+
+  return (
+    <AlertDialog open={open}>
+      <AlertDialogContent>
+        <AlertDialogHeader>
+          <AlertDialogTitle>Token transfer</AlertDialogTitle>
+          <div className="w-full  flex justify-around">
```

Comment:
There are some alignment issues here. Any time the name is longer than the image width, they're noticeable. I noticed it with `myproducer`. I exaggerated it via devtools to better see what was happening: <img width="536" alt="image" src="https://github.com/user-attachments/assets/bdf94222-3a60-462a-a53a-1279ee437592" />